app/src/main/java/com/infomaniak/swisstransfer/ui/screen/onboarding/OnboardingScreen.kt

```diff
@@ -68,7 +70,11 @@ fun OnboardingScreen(goToMainActivity: () -> Unit) {
     }
 
     val onboardingPages = buildList {
-        Page.entries.forEach { page -> add(page.toOnboardingPage(isHighlighted)) }
+        Page.entries.forEachIndexed { index, page ->
+            val isPlayingLottie by remember { derivedStateOf { pagerState.currentPage == index } }
+
+            add(page.toOnboardingPage(isHighlighted, isPlayingLottie = { isPlayingLottie }))
```

Comment:
Maybe we should put the isPlayingLottie logic inside toOnboardingPage as it's part of the creating the OnboardingPage UI